pyproject.toml

```diff
@@ -15,7 +15,7 @@
 [project]
 dynamic = ["version"]
 name = "holoscan-cli"
-requires-python = ">=3.9,<3.13"
+requires-python = ">=3.10,<3.13"
```

Comment:
To allow this to work for 3.13:

Can we change the entry for coveralls [at this line below](https://github.com/nvidia-holoscan/holoscan-cli/blob/b1caa27da169398df0c221cedcddd970dcb7804f/pyproject.toml#L83) to use:[environment markers](https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies) to specify it is only a test dependency if using Python <3.13?

```py
coveralls = { version = "^4.0.1", markers = "python_version < '3.13'" }
```

Reply:
I saw in [.github/workflows/main.yaml](https://github.com/nvidia-holoscan/holoscan-cli/pull/61/files#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3) that coveralls is only actually being run for a single Python version (3.12) so I think that may work  to unblock 3.13 support here?

Reply:
Thanks! I added it to the `tool.poetry.group.test.dependencies` section.